src/Console/Command/ProcessCommand.php

```diff
@@ -135,9 +135,6 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $processResult = $this->processResultFactory->create($files);
         $outputFormatter->report($processResult, $configuration);
 
-        // invalidate affected files
-        $this->invalidateCacheChangedFiles($processResult);
-
```

Comment:
Where are the changed files invalidated now?

Reply:
Like I said, we don't need to invalidate files, we override them, same does phpstan.